src/App.js

```diff
@@ -1,91 +1,128 @@
 import React from 'react';
-import InputBar from './components/InputBar';
-import TodoBox from './components/TodoBox';
+
 import Tabs from './components/Tabs';
+import InputBar from './components/InputBar';
+import TodoList from './components/TodoList';
+
 import './assets/css/';
 
 class App extends React.Component {
   constructor(props) {
     super(props);
 
-    const storageData = window.localStorage.getItem('todoData');
-    const todos = storageData ? JSON.parse(storageData) : [];
-
     this.state = {
-      todos: todos,
+      todos: [],
       tab: -1,
       edit: null,
-      editIndex: null
+      editIndex: null,
+      search: '',
     };
 
     this.startEdit = false;
   }
 
+  /**
+   * add or edit todo item
+   * @param {string} todo
+   * @returns {undefined}
+   */
   addTodo = (todo) => {
-    const todos = this.state.todos.slice();
-
+    let todos = this.state.todos.slice();
     const date = new Date().toLocaleString();
```

Comment:
``` const currentDate = new Date().toLocalString() ```;

Lets avoid naming the variable like date. Its nearly looks like keyword of js and also its too generic.